src/components/common/Button.tsx

```diff
@@ -25,7 +27,7 @@ export function Button({
           : "bg-transparent"
       } ${
         className || ""
-      } flex w-max text-sm  border border-snapBorder hover:border-snapLink p-3 rounded-full justify-center font-normal   focus:outline-none focus-visible:ring-2 active:bg-neutral-800  
+      } flex w-max text-sm  border border-snapBorder hover:border-snapLink ${padding} rounded-full justify-center font-normal   focus:outline-none focus-visible:ring-2 active:bg-neutral-800  
```

Comment:
ideally dont do it this way , just pass the class you want through the className prop with ! to override , so for example className="!p-4"

Reply:
Ah - was having issues overriding. Didn't know you could do that! Ty